core/src/test/java/com/google/cloud/sql/core/CloudSqlInstanceConcurrencyTest.java

```diff
@@ -0,0 +1,351 @@
+package com.google.cloud.sql.core;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.cloud.sql.AuthType;
+import com.google.cloud.sql.CredentialFactory;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.util.concurrent.Futures;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.ListeningScheduledExecutorService;
+import com.google.common.util.concurrent.MoreExecutors;
+import dev.failsafe.RateLimiter;
+import java.io.IOException;
+import java.security.KeyPair;
+import java.sql.Date;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Formatter;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CloudSqlInstanceConcurrencyTest {
+  private static final long start = System.currentTimeMillis();
+
+  private static final Logger logger =
+      Logger.getLogger(CloudSqlInstanceConcurrencyTest.class.getName());
+
+  static {
+    ConsoleHandler handler =
```

Comment:
Is this a dev artifact?

Reply:
I think you're looking for this: https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/main/pom.xml#L197

Reply:
I removed this. I'll use the logging file instead.